plugins/qcheck-stm/test/dune

```diff
@@ -98,6 +105,26 @@
    (diff hashtbl_errors.expected hashtbl_errors)
    (diff hashtbl_stm_tests.expected.ml hashtbl_stm_tests.ml))))
 
+(rule
+ (target testing_warnings_errors)
```

Comment:
I suggest `all_warnings` rather than `testing_warnings` for the name of the test.